src/components/bullet.py

```diff
@@ -13,6 +13,9 @@ class Bullet(AbstractActor):
     def update(self) -> None:
         self.controller.update()
 
+    def collision_check(self) -> bool:
```

Comment:
I don't think this needs to be here since we're not implementing collision detection yet